### ISSUE TITLE
Downloading VSCode doesn't work

### ISSUE THREAD

**Issue by mpancia** [mitchellurgero/cloudron-vscode#2]
I think that the current `download.sh` script doesn't work, as the `code-server` location has moved -- the `curl` command pulls a redirect page. If you add the `-L` flag to the curl line, it'll follow the redirect and correctly download vscode.

**Comment by mpancia** [comment]
https://github.com/mitchellurgero/cloudron-vscode/blob/15cd5f9acf077b12dd81dcc41f7ee067477ae09b/download.sh#L5 This is the offending line, AFAICT.

**Comment by mitchellurgero** [comment]
Hello! Yes I have a working version of this locally right now. However, cloudron changed how they do their Reverse proxy which causes issues in code-server. Once I can work around that, this will be updated.

Thank you for the information though! Glad to see someone is using or trying to use it.

**Comment by mpancia** [comment]
Cool, thanks for making this; I'll be glad to use it when it's working again. :)

**Comment by mitchellurgero** [comment]
Absolutely! Thanks for trying it out! If you downgrade (or install) Cloudron Box server version 3.4.4 or lower it should still work.
